changelog/fix_anchor_for_cops_documentation_generator.md

```diff
@@ -0,0 +1 @@
+* [#12994](https://github.com/rubocop/rubocop/pull/12994): Fix anchor if it contains two or more slashes. ([@ksss][])
```

Comment:
Changelog entry is not necessary due to internal modification. Can you remove it?